frontend/src/scenes/organizationLogic.tsx

```diff
@@ -20,18 +26,46 @@ export const organizationLogic = kea<organizationLogicType<OrganizationType, Per
                         return null
                     }
                 },
-                createOrganization: async (name: string) => await api.create('api/organizations/', { name }),
+                createOrganization: async (name: string) => {
+                    const result = await api.create('api/organizations/', { name })
+                    teamLogic.actions.loadCurrentTeam()
+                    userLogic.actions.loadUser()
+                    return result
+                },
                 updateOrganization: async (payload: OrganizationUpdatePayload) =>
                     await api.update('api/organizations/@current', payload),
+                renameCurrentOrganization: async (newName: string) => {
+                    if (!values.currentOrganization) {
+                        throw new Error('Current organization has not been loaded yet, so it cannot be renamed!')
+                    }
+                    const renamedOrganization = (await api.update(
+                        `api/organizations/${values.currentOrganization.id}`,
+                        {
+                            name: newName,
+                        }
+                    )) as OrganizationType
+                    userLogic.actions.loadUser()
+                    return renamedOrganization
+                },
                 completeOnboarding: async () => await api.create('api/organizations/@current/onboarding/', {}),
             },
         ],
-    },
-    listeners: {
+    }),
+    listeners: ({ values }) => ({
         createOrganizationSuccess: () => {
             window.location.href = '/organization/members'
         },
-    },
+        renameCurrentOrganizationSuccess: () => {
+            toast.success('Organization has been renamed')
```

Comment:
Maybe it should be a larger question than the scope of this PR, but not showing an error message when something went wrong is not good UX.

Reply:
Ah, previously we relied on automatic API error toasts, but these got slashed and now lots of places in the UI are confusing because there's no info when something goes wrong…

Reply:
Maybe we need to start doing case-by-case error handling? Not entirely sure about the best approach here

Reply:
This is kind of pending #3525 and our overall approach to errors. I'm still for blanket catch-and-notify, unsure what (and whether) to do here.